src/aiortc/rtcrtpparameters.py

```diff
@@ -1,48 +1,47 @@
 from collections import OrderedDict
-from typing import List  # noqa
+from dataclasses import dataclass, field
+from typing import List, Optional  # noqa
```

Comment:
The "noqa" and any similar ones can go. The reason it was there was to silence flake8 since the import wasn't actually referenced in code.

Reply:
Remove "noqa" comment, because of `List` and `Optional` really used in this module.